styles/style.scss

```diff
@@ -0,0 +1,12 @@
+// BASE
+@import "./base/reset";
+@import "./base/root";
+
```

Comment:
Notice the order of the sections here:
BASE
then COMPONENTS
then LAYOUT

this is because CSS reads from top to bottom. If we call a function/mixin/variable before its initialized, our SCSS compiler will throw an error!